src/executors/default.yml

```diff
@@ -3,4 +3,5 @@ description: >
   
 resource_class: small
 docker:
-  - image: circleci/buildpack-deps:buster-curl
\ No newline at end of file
+  - image: circleci/buildpack-deps:buster-curl
```

Comment:
I'm unclear on what this change was or if it was necessary.  Did you just add a couple of newlines?  Could/should this change be reverted?

Reply:
Adds new line to eof